src/CMakeLists.txt

```diff
@@ -176,7 +177,11 @@ ENDIF(Qt5WebEngineWidgets_LIBRARIES)
 
 SET(Canorus_Core_MOCs # MOCs compiled into scripting library as well
 	import/import.h
+    import/lilypondimport.h
```

Comment:
Could you please set in QtCreator to use tabs instead of spaces for indentation for consistency with existing source?

Reply:
https://github.com/canorusmusic/canorus/issues/130